packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -27,7 +27,7 @@
 
 .edit-post-visual-editor .block-editor-writing-flow__click-redirect {
 	// Allow the page to be scrolled with the last block in the middle.
-	min-height: 50vh;
+	min-height: 40vh;
```

Comment:
This feels like a random number used to give some space for meta boxes. I'm not sure if we can do better though.

Reply:
We can, but not without a biggish refactor of the component. There's an argument that it was a bit random before as well — 50vh is half of the entire viewport. But it should have been half of the viewport, minus header, footer and optionally the adminbar if not fullscreen, with extra rules for mobile contexts.